src/foam/core/stdlib.js

```diff
@@ -538,7 +538,13 @@ foam.LIB({
     {
       name: 'clamp',
       code: function(min, value, max) {
-        return Math.min(Math.max(value, min), max);
+        if ( min != null && max == null )
```

Comment:
When/how would this be used?

Reply:
The undefined min-max was needed in this PR https://github.com/kgrgreer/foam3/pull/627.  
This function was initially added when we added pagination to tables.